src/MicrosoftTeams.schema.json

```diff
@@ -209,6 +223,14 @@
                         "description": "This value describes whether or not the bot utilizes a user hint to add the bot to a specific channel.",
                         "default": false
                     },
+                    "botType": {
```

Comment:
Gunjan added just a boolean flag for this. Can y'all sync on the right thing to do here?